fastlane/lib/fastlane.rb

```diff
@@ -19,6 +19,7 @@
 require 'fastlane/documentation/docs_generator'
 require 'fastlane/other_action'
 require 'fastlane/plugin_manager'
+require 'fastlane/plugins/plugins'
```

Comment:
Perfect, in the mean-time I actually have more classes already, which I'll be moving into the subfolder after merging in your changes.